clash-prelude/src/Clash/Sized/RTree.hs

```diff
@@ -537,3 +545,125 @@ lazyT :: KnownNat d
       => RTree d a
       -> RTree d a
 lazyT = tzipWith (flip const) (trepeat ())
+-- | Extract the first element of an RTree
```

Comment:
I can't let it go :P. Enter pls.

Reply:
Thanks for spotting that! I reordered the stuff a bit and introduced that omission. Or omitted that introduction, your pick.